jetty-websocket/websocket-common/src/main/java/org/eclipse/jetty/websocket/common/io/FrameFlusher.java

```diff
@@ -334,52 +342,63 @@ public void close()
 
     public void enqueue(Frame frame, WriteCallback callback, BatchMode batchMode)
     {
-        if (closed.get())
```

Comment:
This looks like a race. Testing atomic variable and then conditionally grabbing a lock to act.